presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,163 @@
+# Release 341 (XX Aug 2020)
+
+## General Changes
+
+* Add support for variable-precision `TIME` type. ({issue}`4381`)
+* Add support for variable precision `TIME WITH TIME ZONE` type. ({issue}`4905`)
+* Add Iceberg connector.
+* Add {func}`human_readable_seconds` function. ({issue}`4344`)
+* Add [`reverse`](function_reverse_varbinary) function for `VARBINARY`. ({issue}`4741`)
+* Add support for {func}`extract` for `timestamp(p) with time zone` with values of `p` other than 3. ({issue}`4867`)
+* Add support for correlated subqueries in recursive queries. ({issue}`4877`)
+* Add [](optimizer_rule_stats) system table. ({issue}`4659`)
+* Remove support for political time zones in `TIME WITH TIME ZONE` type. ({issue}`191`)
+* Report dynamic filters stats. ({issue}`4440`)
```

Comment:
Should we abbreviate "statistics" in this context. I think it is fine in config param/function/table name. But i would keep whole word in a description sentence.

Reply:
Fixed